engines/osxnative/config-schema.yml

```diff
@@ -0,0 +1,37 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              Config
+description: |-
+  Config applicable to Mac OSX native engine
+type:               object
+properties:
+  workerGroup:
+    title:          Worker Group
```

Comment:
All of these should be in a global worker config regardless of the engine that is running.